lib/matplotlib/offsetbox.py

```diff
@@ -569,14 +570,16 @@ class DrawingArea(OffsetBox):
     """
     The DrawingArea can contain any Artist as a child. The DrawingArea
     has a fixed width and height. The position of children relative to
-    the parent is fixed.
+    the parent is fixed. The children can be  clipped at the
```

Comment:
Sorry, it looks like I won the race condition on that and merged the previous version.